chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SearchRequest.java

```diff
@@ -144,4 +147,62 @@ public Boolean getHasHadSurveillance() {
 	public void setHasHadSurveillance(Boolean hasHadSurveillance) {
 		this.hasHadSurveillance = hasHadSurveillance;
 	}
+	
+	@Override
+	public int hashCode() {

```

Comment:
Why override hashCode? Also, is the algorithm you're using below some sort of standard kind of algorithm for generating a hashcode?